buildenv/jenkins/common/build.groovy

```diff
@@ -531,6 +531,9 @@ def archive_diagnostics(javaVersionJdkImageDir = null) {
     if (SPEC.contains('win')) {
         // collect *.pdb files to help diagnose omr_ddrgen failures
         findCrashDataCmd += " -o -name '*.pdb'"
+    } else if (SPEC.contains('gcc11')) {
```

Comment:
Shouldn't we collect .debuginfo for all linux specs rather than the special and temporary aarch64_linux_gcc11 spec?

Reply:
Updated to apply for all linux specs.